.travis.yml

```diff
@@ -0,0 +1,24 @@
+# @Date:   2017-03-20T16:36:18-04:00
+# @Project: scatt-jacc-caj
+# @Filename: travis.yml
+# @Last modified time: 2017-03-20T16:53:05-04:00
+
+#defining the language used
+language: java
+
+script:
+#installing dependencies
+- ./gradlew assemble
+#calling default script
+- ./gradlew check

```

Comment:
I just looked at what each of the gradle commands above does when I run it, and `gradle build` does the same thing as running both of the above. Let's change it to gradle build and then I'll be ready to merge this.